container/root/run.sh

```diff
@@ -3,7 +3,7 @@
 if [[ -f /root/.composer/config.json ]]
 then
   echo "Running `composer install`"
-  composer install
+  singularity_runner test
```

Comment:
the `test` command runs the RC file as is. We can define a custom block in the RC file called `docker_build` or something, and the command can be changed to `singularity_runner batch docker_build` instead.

Your preference. See the other PR though for an alternative that doesn't require the runner though.